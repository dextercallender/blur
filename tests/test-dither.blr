void main() {
	char[][] ditherArr = dither("images/leaf.jpg");
}
