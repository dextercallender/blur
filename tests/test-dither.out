$
50
50
$$$$$$$$$$$$$$$$@$$@$@@@@$@@@@$@@@$@@@@$@$@@@$@@$$
$$$$$$$$$$$$$$$$$@@@$@@@@@@@$$@@$@@$$@@@@@@@@@@@$$
$$$$$$$$$$$$$$$$@@@$$@@@@@@$@@@@@@@@@8kQJt(]k@@$$$
$$$$$$$$$$$$$$$$@$@$$@@@@@$@@$$$oZXt]+_+++(8$@@@$$
$$$$$$$$$$$$$$$$@@@@$@$@$@@@MQr]++++__+++(8@@$$$$$
$$$$$$$$$$$$$$$$$@@@@@$@$oX{++__++++++++(8@@@@@$$$
$$$$$$$$$$$$$$$$$@@$@@@Z(_+++++++++_+++?8$@$@@@@$$
$$$$$$$$$$$$$$$$$@@$8J]+++_++___++_+_++k$@$@@@@@$$
$$$$$$$$@@@$@$$$@@8J]+++++++++_+++++++X$$$$$$$$$$$
$$$$$$$$@@@@@$$$@q{+++++++_+++_++_+++]8@$$$$$$$$$$
$$$$$$$$@@@$$@@8r++_+_+++__{_+++++++_Z$@$$$$$$$$$$
$$$$$$$$@@@@@@o(+__+_++++vX++_+++++_{$@$$$$$$$$$$$
$$$$$$$$$@$@@o?+++++++_tov+_++++_+++Z@@@$$$$$$$$$$
$$$$$$$$@@$@o?_+++_++]QM(++__++++++]8@@@$$$$$$$$$$
$$$$$$$$$@@8(++_+++_{ao?++++_++++++J@$@@$$$$$$$$$$
$$$$$$$$@@@u+++++_+(8o]+_+++++_+++_M@$$@$$$$$$$$$$
$$$$$$$$$@b++__+++(8o]++++__+++_++r@@@@@$$$$$$$$$$
$$$$$$$$$@{+_++++(8o?++_+++__+++++k$@@@@$$$$$$$$$$
$$$$$$$$@q+_++++(8M?+__+_+_+++++_{@@@$$$$$$$$$$$$$
$$$$$$$$@t+_+++?o8(+++_+__+__++++Q@$$@$@$$$$$$$$$$
$$$$$$$$M++++++k@x+++_+++_+_~+_+?8@@$@$@$$$$$$$$$$
$$$$$$$$Z+_+_+J@Q+++++++++++++++X@@@@$$$$$$$$$$$$$
$$$$$$$$u_+_+t@o+++_+_++++++_++_M$@@@@$@$$$$$$$$$$
$$$$$$$$(+_+]M@(_+++++++++++++_u$$@@@@$@$$$$$$$$$$
$$$$$$$$++++Q@J+++++++++++++++]8$$$$$$$$$$$$$$$$$$
$$$$$$$$+++t@M]++++++++++_+_++J@$$$$$$$$$$$$$$$$$$
$$$$$$$$+++M$u++++++++++_+_++(@@$$$$$$$$$$$$$$$$$$
$$$$$$$$++J$M+++++++++++++_+?M@@$$$$$$$$$$$$$$$$$$
$$$$$$$$+?8@v+_++++++++++++_k@@@$$$$$$$$$$$$$$$$$$
$$$$$$$$+Z@M+_+_++++++++_+_Z$@@@$$$$$$$$$$$$$$$$$$
$$$$$$$$t@@X+++++++++++++]b@@@@@$$$$$$$$$$$$$$$$$$
$$$$$$$$8@@]++++++++++++tM@@$@$$$$$$$$$$$$$$$$$$$$
$$$$$$$$@@q+++_+++_+++{Z$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$$@r+__+_++++uk@@$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$$@]+++_++xZ8@$@@$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$$k++_(uq8$@@@$@@$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$@J_+(@@@@@@@@$$@$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$@t+_J@$$@@@@$@@@$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$@]++b@@@$$@$@$@$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$M+++M$$$@$@@@@@$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$k_++$@$@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$Q++|@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$U+_t@$@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$J++U@@@@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$X+_J$@@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$t++L@@$@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$t+_k@$$@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$t_+k$@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$J+(8@@@$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
